packages/lesswrong/components/tagging/LWTagPage.tsx

```diff
@@ -1011,7 +1012,7 @@ const LWTagPage = () => {
     tagName={tag.name}
     tagId={tag._id}
     sortedBy={query.sortedBy || "relevance"}
-    limit={terms.limit}
+    limit={terms?.limit ?? 0}
```

Comment:
I'm not sure what the behavior of a limit of 0 is, but it wouldn't totally shock me if it were something like "no limit"

Reply:
This limit is actually going to an `AnalyticsContext`, where it can do no harm. The `terms.limit` going into `PostsList2`, where it winds up in a query, is behind a truthiness check.